cloud/pkg/admissioncontroller/admission.go

```diff
@@ -98,6 +99,8 @@ func Run(opt *options.AdmissionOptions) error {
 	http.HandleFunc("/rules", serveRule)
 	http.HandleFunc("/ruleendpoints", serveRuleEndpoint)
 	http.HandleFunc("/offlinemigration", serveOfflineMigration)
+	http.Handle("/readyz/", http.StripPrefix("/readyz/", &healthz.Handler{}))
```

Comment:
why  we  need StripPrefix?

Reply:
nothing, just ref: https://github.com/karmada-io/karmada/blob/ed9b838056ee45720abf51b2d603548e6ce922bf/cmd/webhook/app/webhook.go#L125